movies/src/resources/doc.jsonld

```diff
@@ -43,6 +43,16 @@
           "property": "schema:name",
           "title": "name",
           "description": "The movie's name"
+        },
+        {
+          "property": "schema:description",
+          "title": "description",
+          "description": "Textual description of the movie."
+        },
+        {
+          "property": "schema:duration",
+          "title": "duration",
+          "description": "The movie's duration in ISO 8601 date format."
```

Comment:
Oh, this makes me realise. We need at least ranges for each property.

Reply:
Shouldn't the vocabulary provide the range? What would it be here?

Reply:
And why exactly do we need it?

Reply:
> Shouldn't the vocabulary provide the range?

Maybe, maybe not. `schema:duration` actually does not define a `rdfs:range`. And besides, an API can be more specific. Say, a property has range `xsd:Integer`. A server could narrow it down to `xsd:PositiveInteger`. 

And then even if it's defined by the vocabulary, would you have the client (browser?) dereference `schema:duration` to find what is the datatype to use?

That information should be explicit.

> And why exactly do we need it?

To be able to build an accurate UI

Reply:
I thought we're going with baby steps. Indeed, the more details on possible values, the better, but do we need it here and now? We could use SHACL for that, but I don't think it is really needed her at the moment.

Reply:
> Indeed, the more details on possible values, the better, but do we need it here and now?

Sure, let's do more smaller PRs. But, the RDFS ranges are IMO the minimum requirement for a supported property. Without them the client has only an option to fill the operation request with plain literals. It has nothing to do with more complex scenarios or SHACL.

Reply:
> But, the RDFS ranges are IMO the minimum requirement for a supported property

I'm keen to agree with you, but having more details explicitly available is something an API may not provide.

> Without them the client has only an option to fill the operation request with plain literals.

Untrue - knowing a property IRI client could i.e. de-reference it hoping to receive more details on that property, including it's range/domain declarations. It's linked data after all